src/ResourceManager/KeyVault/Commands.KeyVault.Test/packages.config

```diff
@@ -4,20 +4,21 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
-  <package id="Microsoft.Azure.KeyVault" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Authorization" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.KeyVault" version="1.0.1" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault" version="2.0.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault.WebKey" version="2.0.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.KeyVault" version="2.0.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.Framework" version="1.0.6052.28118-prerelease" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.7-preview" targetFramework="net45" />
-  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
```

Comment:
Microsoft.Azure.Management.Authorization 2.0.0 -> Azure Common 2.1.0

We need Microsoft.Azure.Management.Authorization otherwise tests fail with - The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: TestManagementClientHelper class wasn't initialized with the AuthorizationManagementClient client.